backend/src/main/java/pt/ulisboa/tecnico/socialsoftware/tutor/fraud/dto/QuizFraudScoreDto.java

```diff
@@ -0,0 +1,23 @@
+package pt.ulisboa.tecnico.socialsoftware.tutor.fraud.dto;
+
+import java.io.Serializable;
+
+public class QuizFraudScoreDto implements Serializable{
```

Comment:
QuizFraudTimeScoreDto

Reply:
This is the Building DTO Object for a fraud entry, consisting of a user and a score. I'll add another class for the specialization of a QuizFraudTimeScoreDto and rename this to UserFraudScoreDto